internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1674,7 +1677,15 @@ components:
           description: flag indicating whether the capacity for the instance type
             in the region is reached
           type: boolean
+        available_sizes:
+          description: list of available Kafka instance sizes that can be created
```

Comment:
maybe add some note about current limits, e.g. `list of available Kafka instance sizes that can be created when taking into account current capacity and limits`. WDYT?

Reply:
Description has now been updated to note the limits